dtool/src/interrogate/interrogateBuilder.cxx

```diff
@@ -1696,8 +1696,6 @@ get_function(CPPInstance *function, string description,
     InterrogateFunction &ifunction =
       InterrogateDatabase::get_ptr()->update_function(index);
 
-    nassertr(&ifunction != NULL, 0);
```

Comment:
This is a case where it is never supposed to return a null reference to begin with, and the assert was just added to save me a lot of time in the debugger in case it happened again.

I agree that this is kind of smelly and we could probably improve the InterrogateDatabase interface altogether (though I don't really think it's high priority).

Reply:
Resolution: remove